src/components/shared/MarkerTooltipContents.js

```diff
@@ -11,6 +11,19 @@ import formatTimeLength from '../../utils/format-time-length';
 import type { TracingMarker } from '../../types/profile-derived';
 import type { MarkerPayload } from '../../types/markers';
 
+function markerDetail(key, label, value) {
```

Comment:
The Minor GC count was easy, but it could be useful.  Is that what total_zones means, the actual total in the JS runtime and not the total that may have been "considered" for this GC or something weird, like total_zones were marked but only zones were swept..

I'll try to add the units, for the "# MInor GCs" etc, I like to show that it's a count because that's the units it's measured in. Maybe "Nu. Minor GCs"?